packages/ui/src/components/TreeView/TreeView.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import Card from "../Card";
+
+import "./tree-view.css";
+
+interface TreeViewItemProps {
+  noBranch?: boolean;
+  children: React.ReactNode;
+}
+
+const Item = ({ noBranch, children, ...rest }: TreeViewItemProps) => (
+  <div
+    className="tree-view-item"
+    data-h2-margin-left="base(x2)"
+    data-h2-padding-left="base(x1)"
+    data-h2-padding-top="base(x1)"
+    data-h2-border-left="base:selectors[:not(:last-child)](1px solid gray.light)"
+    data-h2-content="base:selectors[::before]('')"
+    data-h2-display="base:selectors[::before](block)"
+    data-h2-position="base:selectors[::before](absolute)"
```

Comment:
You can double up properties here to remove the css completelty. This should work:

```suggestion
    data-h2-position="base(relative) base:selectors[::before](absolute)"
```